product_docs/docs/pgd/5/other_considerations.mdx

```diff
@@ -4,33 +4,33 @@ title: "Other considerations"
 
 Review these other considerations when planning your deployment.
 
-## Data Consistency
+## Data consistency
 
 Read about [Conflicts](consistency/conflicts/) to understand the implications of the asynchronous operation mode in terms of data consistency.
 
 ## Deployment
 
-EDB PGD is intended to be deployed in one of a small number of known-good configurations, using either [Trusted Postgres Architect](/tpa/latest) or a configuration management approach and deployment architecture approved by Technical Support.
+PGD is intended to be deployed in one of a small number of known-good configurations, using either [Trusted Postgres Architect](/tpa/latest) or a configuration management approach and deployment architecture approved by Technical Support.
```

Comment:
EDB PGD - it's a product reference.

Reply:
> EDB PGD - it's a product reference.

Right, we don't need EDB when using the short name